src/stories/Library/availability-label/availability-label.scss

```diff
@@ -32,4 +32,11 @@
       display: inline-block;
     }
   }
+
+  & p {
```

Comment:
Please add a new element within the availability label e.g. `.availability-label__text` or `.availability-label__type` and `.availability-label__status` instead of targetting the generic `<p>`.